### ISSUE TITLE
Can't generate qrcode from numbers

### ISSUE THREAD

**Issue by jenniferki** [Cordobo/angularx-qrcode#92]
Hello,

I am not able to generate the code if I use numbers.
Example:             
<qrcode [qrdata]="88" [width]="500" [errorCorrectionLevel]="'H'" [elementType]="'svg'"></qrcode>
Result: angularx-qrcode.js:171 [angularx-qrcode] error:  Error: Invalid data
or this one:
<qrcode [qrdata]="pumpID" [width]="500" [errorCorrectionLevel]="'H'" [elementType]="'svg'"></qrcode>
where pumpID is declared the following way:   public pumpID: string;
if there's a number in pumpID it will fail with the same error although the variable is of string type.

Please help.

**Comment by Cordobo** [comment]
Hi @jenniferki 

I'm not sure if I got you right.

So I assume you have a constant named `pumpId` typed as union type (string | number), which can hold any string or int/float:

`let pumpId: string | number`

If you pass this the constant `pumpId` to the property `qrdata`, TypeScript complains, that the both types are not compatible as it expects a string type only

If you make sure, that you always pass a string like so, it should work:

```
const sampleDataSource: string | number = 88

const pumpId: string = String(sampleDataSource)
const pumpId: string = sampleDataSource.toString()
const pumpId: string = `${sampleDataSource}`
```

**Comment by jenniferki** [comment]
Hi,

pumpID is of type string so I wasn't sure why the qrdata wasn't accepting it, but then I saw that it gets there as a number (88 instead of "88") so I just converted it to string using [qrdata]="pumpID.toString()" and it worked.

There is another thing about the new version for Angular 7. Currently we are using version 1.5.3 with Angular 7 project and we have another place where we generate a qrcode. The user types double digit number in a textbox and presses ok to generate the qrcode. The textbox control name is tubeLabelCode. In version 1.5.3 the following works fine [qrdata]="tubeLabelCode.value", but when I tried to upgrade to version 2.0.0 I got the same problem as with the pumpID and I can't use toString() on it since when the textbox is empty the qr throws an error. So I had to revert back to version 1.5.3
Any idea why this works in the old version and not in the new one?

Thanks,
Jennifer
